pkg/util/validation/formats.go

```diff
@@ -261,6 +262,22 @@ func (FormatTimezone) CheckFormat(value interface{}) error {
 	return nil
 }
 
+type FormatDateTime struct{}
+
+func (FormatDateTime) CheckFormat(value interface{}) error {
```

Comment:
Include some tests for the new format.
- With integer second
- With some milliseconds
- With UTC timezone
- With some explicit timezone offset

Reply:
Added some tests, thanks!